src/lib/three-d-secure.js

```diff
@@ -54,6 +54,7 @@ ThreeDSecure.prototype.verify = function (payload, merchantProvidedData) {
 
   verifyOptions.additionalInformation = verifyOptions.additionalInformation || {};
   verifyOptions.additionalInformation.acsWindowSize = verifyOptions.additionalInformation.acsWindowSize || DEFAULT_ACS_WINDOW_SIZE;
+  this._model.verifyCardReady();
```

Comment:
I don't think this is the correct place to call `verifyCardReady`. This is happening when lookup completes, but that's not taking into account when step up authentication occurs (that happens in the verifyCard function)

Reply:
I think this logic is going to be a little more complex to cover both scenarios where 3DS "completed" because it didn't require step up authentication, and where 3DS completes after step up authentication. Happy to talk through this in person, let me know!

Reply:
I believe it's being called here to toggle `this._shouldWaitForVerifyCard` back to false. So after it's complete, we should no longer wait for verification. Maybe a clearer way to use this function is to rename it and explicitly pass in a boolean value?

Reply:
Pardon my mobile brain but is there a reason we need a method at all vs setting the property directly? This may be a JS thing so please excuse my very basic questions!

Reply:
@jaxdesmarais this is a great question and after discussion, we have refactored this to set the property directly.